astroquery/mpc/core.py

```diff
@@ -1,13 +1,17 @@
 # -*- coding: utf-8 -*-
 
-from bs4 import BeautifulSoup
+import json
+import re
+import warnings
 
 import numpy as np
+from bs4 import BeautifulSoup
 from astropy.io import ascii
 from astropy.time import Time
-from astropy.table import Table, Column
+from astropy.table import Table, QTable, Column
 import astropy.units as u
-from astropy.coordinates import EarthLocation, Angle
+from astropy.coordinates import EarthLocation, Angle, SkyCoord
+from astropy._erfa.core import ErfaWarning
```

Comment:
this usage here prompted me to make the ErfaWarning exposed somewhere directly, so it's not available from astropy.utils.exceptions in 2.0.12 and 3.1.2. Given that we still support older ones here a try except will be needed for a while, but it's preferred to not use anything directly from `_erfa`

Reply:
So you mean something like this:

```
try:
    from astropy.utils.exceptions import ErfaWarning
except ImportError:
    from astropy._erfa.core import ErfaWarning
```
? Or do you want me to get rid of `astropy._erfa` entirely?

Reply:
sorry, I didn't notice this reply. I'm afraid yes, we need to do the try except as the warning became available in `astropy.utils.exceptions` very recently in 2.0.12 (triggered by this PR). Maybe add a line of comment before the _erfa import that this usage is discouraged and should be removed once we require 2.0.12/3.1.2+ in astroquery.

Alternatively we could go bold and ignore all `AstropyUserWarnings`, though I'm not sure that's a wise choice.